test/integration/forms/form.js

```diff
@@ -1334,6 +1347,8 @@ context('Patient Action Form', function() {
         expect(data.attributes.response.data.patient.fields.weight).to.equal(192);
       });
 
+    cy.wait(6000);
```

Comment:
so a `6000` wait is super long.  there's a `cy.clock()` where we can force that to happen.  If you search for `clock()` I think you can find examples